src/client/game.cpp

```diff
@@ -3968,7 +3982,48 @@ inline void Game::updateProfilerGraphs(ProfilerGraph *graph)
 	graph->put(values);
 }
 
+/****************************************************************************
+ * Shadows
+ *****************************************************************************/
+void Game::updateShadows(float _timeoftheday)
+{
+	ShadowRenderer *shadow = RenderingEngine::get_shadow_renderer();
+	if (shadow) {
```

Comment:
invert this condition to return, this will prevent a whole function block indent

Reply:
Done.